lang/en/docs/cli/actions/create-python-env.md

```diff
@@ -2,19 +2,21 @@
 
 Here, we explain how to use the **"virtualenv"** command [^1] within the [Command Line Interface](../overview.md) (CLI) to assemble some commonly used python packages for materials science computations into a **virtual environment**. These packages include the **"pymatgen"**, **"pandas"** and **"atomic simulation environment (ase)"** libraries.
 
-!!! note "Default version of python is 2.7.5"
-    By default the system-wide version of python is used.
+!!! note "Default version of Python is 2.7.5"
+    By default, the system-wide Python 2.7.5 is available.  Other versions of Python are available for use; see [the Command Line Environment documentation's section on Python](../environment.md#default-python-environment) for more details.  
```

Comment:
A user today had some confusion about the meaning of the word "default":

(1) in the modules, the "default" is python 3.8.6 now, so `module load python` loads it
(2) in CLI right after login `python --version` gives 2.7.5

Perhaps we should clarify here too.

Reply:
addressed in https://github.com/Exabyte-io/documentation/pull/181/commits/3a1677e428b8895d79a278d359424894a44cbe91